src/arrayMethodSort.js

```diff
@@ -3,9 +3,42 @@
 /**
  * Implement method Sort
  */
+
+function compareAsString(a, b) {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
```

Comment:
else is also redundant here, just move this `if` check next to the first one

Reply:
Thank you! Fixed it.